stories/components/Card.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import styled from "styled-components";
+import { Card as CardComponent, Text } from "../../src";
+
+export default {
+  title: "Components/Card",
+  component: CardComponent,
+} as ComponentMeta<typeof CardComponent>;
+
+const StyledTtitle = styled(Text)`
+  color: ${({ theme }) => theme.colors.titles};
+  marginBottom: 0.5rem;
+`;
+const StyledText = styled(Text)`
+  color: ${({ theme }) => theme.colors.texts};
+`;
+
+const Template: ComponentStory<typeof CardComponent> = (props) => (
+  <CardComponent {...props}>
+    <StyledTtitle>This is my title</StyledTtitle>
+    <StyledText>{'There are many like it, but this one is yours. You can always return here for a bird\'s-eye view of your holdings. The cards below show the total balance and utilization of your wallet, and the holdings and performance for every Yearn product you use.'}</StyledText>
```

Comment:
nit: Worth to use ticks in order to not have to escape the single quote, I know this is just a story though

```suggestion
    <StyledText>{`There are many like it, but this one is yours. You can always return here for a bird's-eye view of your holdings. The cards below show the total balance and utilization of your wallet, and the holdings and performance for every Yearn product you use.`}</StyledText>
```